src/app/app.component.ts

```diff
@@ -34,6 +37,13 @@ export class AppComponent implements OnInit, AfterViewInit {
           let url = 'https://sparecores.com';
           if(event?.urlAfterRedirects?.length > 1) {
             url += event?.urlAfterRedirects;
+            if(event?.urlAfterRedirects?.startsWith('/og/')) {
+              this.showHeader = false;
+              this.showFooter = false;
+            } else {
+              this.showHeader = true;
+              this.showFooter = true;
+            }
```

Comment:
do we need this branch? this is the default if I get this right
```suggestion
            }
```

Reply:
it cannot hurt

Reply:
I think it can ... if this branch is not there for a good reason, then it adds noise to the codebase and also gives us a false sense of safety. e.g. later someone reviews the code, and decides to drop the defaults on the top, as there's this fallback anyway, but the 3rd branch (`if(event?.urlAfterRedirects?.length < 1)`) is not being handled.

if you feel strong about keeping it, I'm not opposed, but I really don't understand the value of this so double checking